src/main/java/com/magnaboy/CitizensPlugin.java

```diff
@@ -399,6 +393,35 @@ public static void reloadCitizens(CitizensPlugin plugin) {
 		}
 		Util.log("Reloaded Citizens");
 	}
+
+	public void despawnEntity(Entity e) {
+		if (e instanceof Citizen) {
+			citizens.remove((Citizen) e);
+		}
+		if (e instanceof Scenery) {
+			scenery.remove((Scenery) e);
+		}
+		e.despawn();
+	}
+
+	private void cleanup() {
```

Comment:
Nit: I feel like cleanup (and its usages elsewhere) is ambigous, what is it doing? What does .clear() on these do anyway? Is it different from despawning?

Reply:
My goal was to have each class have a cleanup method that we can call on shutdown. The main plugin class has a cleanupAll method that will call all the applicable cleanup methods.

Reply:
That sounds good to me.